core/src/operations/mod.rs

```diff
@@ -441,6 +443,148 @@ pub enum Operation {
     RCombBase,
 }
 
+pub enum OperationData {
+    Felt(Felt),
+    U32(u32),
+    None,
+}
```

Comment:
If we implement, `Serializable`/`Deserializable` for `Operation` as suggested in one of the previous comments, we probably won't need this struct.

Reply:
merged `next`, and removed `OperationData`